business_requirement_deliverable_cost/views/business.xml

```diff
@@ -31,6 +31,9 @@
                                 class="oe_highlight"/>
                     </group>
                 </xpath>
+                <xpath expr="//field[@name='price_total']" position="after">
+                    <field name="gross_profit"/>
```

Comment:
Not only: I would like to see the same fields as in the BR (check screenshot that I have provided).
Besides it should only be visible to the "cost" group